docs/BUILD.md

```diff
@@ -0,0 +1,13 @@
+1. Compile relevant binaries by running the following script:
```

Comment:
[Required]: I'm missing a title/header indicating what is the content of this file